packages/client/src/app/services/utility/search.service.ts

```diff
@@ -1,62 +1,39 @@
 import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
-import { Observable, throwError } from 'rxjs';
-import { catchError, map } from 'rxjs/operators';
+import { Observable } from 'rxjs';
 
 import { InitialResults, PaginateResult } from '@pulp-fiction/models/util';
 import { AlertsService } from '../../shared/alerts';
 import { User } from '@pulp-fiction/models/users';
-import { BlogsContentModel, ContentModel, ProseContent } from '@pulp-fiction/models/content';
+import { BlogsContentModel, ContentModel } from '@pulp-fiction/models/content';
+import { NetworkService } from '../network.service';
 
 @Injectable({
-  providedIn: 'root'
+    providedIn: 'root'
 })
 export class SearchService {
-  private url: string = `/api/search`
+    workResults: Observable<PaginateResult<ContentModel>>;
+    blogResults: Observable<PaginateResult<BlogsContentModel>>;
+    userResults: Observable<PaginateResult<User>>;
```

Comment:
I left this alive for reasons similar to NotificationsService even though it doesn't have any state _yet_.